pkg/daemon/ceph/osd/remove.go

```diff
@@ -138,5 +138,23 @@ func removeOSD(clusterdContext *clusterd.Context, clusterInfo *client.ClusterInf
 		logger.Errorf("failed to remove CRUSH host %q. %v", hostName, err)
 	}
 
+	// The ceph health warning should be silenced by archiving the crash
+	crash, err := client.GetCrash(clusterdContext, clusterInfo)
+	if err != nil {
+		logger.Errorf("failed to list ceph crash. %v", err)
```

Comment:
If there are no crashes to remove, it won't print the error, right?

Reply:
yes fixed.